### PR TITLE
Marked alphanumeric as deprecated

### PR DESCRIPTION
**Pull request by clementh59**
Set the numbers only to be the default
Updated stats, results and trace to not contain the custom path
Added an alert when you verify a deprecated GS1 Digital Link
Removed the possibility to choose your format

### CODE REVIEW

**Review by C-D-Lewis — DISMISSED** [review]

File: src/scripts/verifier.js [hunk 1]
```diff
@@ -17,6 +18,30 @@ const UI = {
 
 const getQueryParam = name => new URLSearchParams(window.location.search).get(name);
 
+const checkIfUriIsDeprecated = (dl) => {
```
Comment (C-D-Lewis) [inline src/scripts/verifier.js:21]: Add JSDoc explaining that if an alpha mapping is present, it is considered deprecated _because_ it should be a number AI instead.

By the way, are alphanumeric aliases like `gtin` actually deprecated in the standard? Or do we just not prefer them?

File: src/scripts/verifier.js [hunk 2]
```diff
@@ -17,6 +18,30 @@ const UI = {
...
+        alert(`Using ${Object.keys(dl.getIdentifier())[0]} is deprecated!`);
+      }
+    });
+
+  if (dl.getKeyQualifiers()) {
+    Object.keys(ALPHA_MAP)
+      .forEach((key) => {
+        Object.keys(dl.getKeyQualifiers())
+          .forEach((keyQualifier) => {
+            console.log(keyQualifier);
```
Comment (C-D-Lewis) [inline src/scripts/verifier.js:34]: Is `console.log` required for functionality?

File: src/scripts/verifier.js [hunk 3]
```diff
@@ -17,6 +18,30 @@ const UI = {
 
 const getQueryParam = name => new URLSearchParams(window.location.search).get(name);
 
+const checkIfUriIsDeprecated = (dl) => {
+  Object.keys(ALPHA_MAP)
+    .forEach((key) => {
+      if (Object.keys(dl.getIdentifier())[0] === ALPHA_MAP[key]) {
+        alert(`Using ${Object.keys(dl.getIdentifier())[0]} is deprecated!`);
```
Comment (C-D-Lewis) [inline src/scripts/verifier.js:25]: I think throwing an error or logging in the console is more appropriate for a deprecation: `alert` blocks _all_ JS on the page, including UI responsiveness.

File: src/scripts/verifier.js [hunk 4]
```diff
@@ -17,6 +18,30 @@ const UI = {
...
+      }
+    });
+
+  if (dl.getKeyQualifiers()) {
+    Object.keys(ALPHA_MAP)
```
Comment (C-D-Lewis) [inline src/scripts/verifier.js:30]: `find` / `filter` / `reduce` could probably do this in a neater fashion, but this is enough for now.

File: src/scripts/generator.js [hunk 5]
```diff
@@ -81,6 +81,9 @@ const updateQrCode = () => {
 const updateDigitalLink = () => {
   digitalLink = DigitalLink();
 
+  // I authorize the key qualifiers to be automatically sorted following the GRAMMAR restrictions
```
Comment (C-D-Lewis) [inline src/scripts/generator.js:84]: I don't understand this comment, can you explain/rephrase?

**Review by C-D-Lewis — CHANGES_REQUESTED** [review]

File: src/data/alpha-map.json [hunk 6]
```diff
@@ -3,13 +3,8 @@
   "8006": "itip",
   "8013": "gmn",
   "8010": "cpid",
-  "410": "shipTo",
```
[the remark this answers is not in the document]
Comment (clementh59) [reply src/data/alpha-map.json:6]: this file is useful only for the alert to say that you use deprecated syntax..

File: src/scripts/verifier.js (hunk 1 above)
Comment (clementh59) [reply src/scripts/verifier.js:25]: Yes they are deprecated

File: src/scripts/verifier.js (hunk 3 above)
Comment (clementh59) [reply src/scripts/verifier.js:29]: I'll do it in another way

File: src/scripts/verifier.js [hunk 7]
```diff
@@ -25,14 +59,32 @@ const onVerifyClicked = () => {
...
+    UI.divResults.innerHTML =
+      Utils.generateResultsHtml(finalUriWithoutCustomPath);
+    UI.divTrace.innerHTML =
+      Utils.generateTraceHtml(finalUriWithoutCustomPath).replace('display mode: ASCII', '');
 
     const isValid = dl.isValid() && dl.getValidationTrace().success;
     UI.spanVerdictResult.innerHTML = `<strong>${isValid ? 'VALID' : 'INVALID'}</strong>`;
     UI.imgVerdict.src = `./assets/${isValid ? '' : 'in'}valid.svg`;
     UI.inputVerifierQuery.value = finalUri;
+    if (hasDeprecatedTerms) {
```
Comment (C-D-Lewis) [inline src/scripts/verifier.js:83]: This can be one line:
```js
UI.divDeprecatedMessage.style.visibility = hasDeprecatedTerms ? 'visible' : 'hidden';
```

**Review by C-D-Lewis — CHANGES_REQUESTED** [review]
One last thing...

**Review by C-D-Lewis — APPROVED** [review]
Good work!

**Pull request merged**